app/assets/geojsons/HG_-1.geojson

```diff
@@ -0,0 +1,205 @@
+
+{
+    "type": "FeatureCollection",
+    "generator": "JOSM",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "male": "yes",
+                "amenity": "toilets",
+                "indoor": "room",
+                "name": "WC-H",
+                "level_name": "-1"
+            },
+            "geometry": {
+                "type": "Polygon",
+                "coordinates": [
+                    [
+                        13.13219416721,
+                        52.39397398336
+                    ],
+                    [
+                        13.13208460674,
+                        52.39396990756
+                    ],
+                    [
+                        13.13207924232,
+                        52.39402146549
+                    ],
+                    [
+                        13.13218842279,
+                        52.39402490812
+                    ],
+                    [
+                        13.13219039206,
+                        52.39401088823
+                    ],
+                    [
+                        13.13219416721,
+                        52.39397398336
+                    ]
+                ]
```

Comment:
1. to see see the rooms one has to add 
```js
  await loadGeoJsonFile("/assets/H_0.geojson");
  await loadGeoJsonFile("/assets/H_1.geojson");
  await loadGeoJsonFile("/assets/H_2.geojson");
  await loadGeoJsonFile("/assets/H_3.geojson");
  await loadGeoJsonFile("/assets/HG_-1.geojson");
  await loadGeoJsonFile("/assets/HS_0.geojson");
```
into `leafletMap.js`.
(or the more performant way)

```js
    await Promise.all(
        loadGeoJsonFile('/assets/ABC-Building-0.geojson'),
        loadGeoJsonFile('/assets/ABC-Building-1.geojson'),
        loadGeoJsonFile('/assets/H_0.geojson'),
        loadGeoJsonFile('/assets/H_1.geojson'),
        loadGeoJsonFile('/assets/H_2.geojson'),
        loadGeoJsonFile('/assets/H_3.geojson'),
        loadGeoJsonFile('/assets/HG_-1.geojson'),
        loadGeoJsonFile('/assets/HS_0.geojson')
    );
```.

Then the js errors with:
```log
leaflet-src.js:1280 Uncaught (in promise) Error: Invalid LatLng object: (undefined, undefined)
    at new LatLng (leaflet-src.js:1280:1)
    at coordsToLatLng (leaflet-src.js:8320:1)
    at coordsToLatLngs (leaflet-src.js:8331:1)
    at coordsToLatLngs (leaflet-src.js:8331:1)
    at geometryToLayer (leaflet-src.js:8289:1)
    at NewClass.addData (leaflet-src.js:8200:1)
    at NewClass.addData (leaflet-src.js:8187:1)
    at NewClass.initialize (leaflet-src.js:8170:1)
    at new NewClass (leaflet-src.js:296:1)
    at Object.geoJSON (leaflet-src.js:8502:1)
```

It seems to me like this is caused by incorrect Polygon coordiantes.
In the (working ABC geojsons there is another array around the coordinates in the polygon):
```suggestion
                "coordinates": [[
                    [
                        13.13219416721,
                        52.39397398336
                    ],
                    [
                        13.13208460674,
                        52.39396990756
                    ],
                    [
                        13.13207924232,
                        52.39402146549
                    ],
                    [
                        13.13218842279,
                        52.39402490812
                    ],
                    [
                        13.13219039206,
                        52.39401088823
                    ],
                    [
                        13.13219416721,
                        52.39397398336
                    ]
                ]]
```

Reply:
In TS:

```ts
// currently:
type Coordinates = [number, number][]
// correct:
type Coordinates = [[number, number][]]

```